### DISCUSSION TITLE
Button LED illumination

### DISCUSSION THREAD

**Discussion by serdarqw** [calibx/msfsdeck#229] [Q&A]
Hi. I've created a profile for MSFS and would like to highlight the programmed buttons with an LED. How can I do this? They are highlighted in the finished profile. Sorry, my English is poor - I use Google.

**Comment by calibx** [comment]
Hi, I think led will be usable with #255. I will work on it after 6.0 compat.
Sorry for the one year wait ... this api doc was not available in the SDK ...
